babel.config.js

```diff
@@ -36,27 +36,17 @@ module.exports = {
     '@babel/preset-react',
     [
       '@babel/preset-typescript',
-      { isTSX: true, allExtensions: true, disallowAmbiguousJSXLike: true }
+      {
+        isTSX: true,
+        allExtensions: true,
+        disallowAmbiguousJSXLike: true,
+        // Svelte script and template parts of a file are handled separately
+        // so Babel can't recognize that an import for a child component was actually being referenced by a source file
+        // because only the script is handed to it and the reference is in the template
+        onlyRemoveTypeImports: true
```

Comment:
I've managed to move this to a single config after fixing import cycles in the action files